test.js

```diff
@@ -119,7 +119,7 @@ test('activeCount and pendingCount properties', async t => {
 	t.is(limit.activeCount, 0);
 	t.is(limit.pendingCount, 1);
 
-	await Promise.resolve();
+	await delay(0);
```

Comment:
This change seems to no longer be required. Tests are passing with `await Promise.resolve();` here. And below. Can you fix?